djangosaml2idp/processors.py

```diff
@@ -11,6 +11,10 @@ def has_access(self, user):
         """
         return True
 
+    def is_enabled(self, request):
```

Comment:
@mhindery re's the merging of is_enabled / and has_access methods, I think you're right - it feels like a better solution to just pass request into has_access, so that is able to do additional checks.

I'd be happy to merge these.

Reply:
Allright, then I'll wait for your updates for the merge.